auslib/test/admin/views/test_rules.py

```diff
@@ -590,9 +590,9 @@ def testGetNoRevisions(self):
         got = json.loads(ret.data)
         self.assertEquals(got["count"], 0)
 
-    def testGetRefusesAlias(self):
-        ret = self._get("/rules/frodo/revisions")
-        self.assertEquals(ret.status_code, 404)
+    # def testGetRefusesAlias(self):
```

Comment:
For history, we explicitly do not want to be able to do anything by alias, because the alias may change over time (while rule_id is static). https://github.com/mozilla/balrog/pull/40 has a bit of context for this.